solvation_analysis/tests/test_solvation_analysis.py

```diff
@@ -77,28 +85,62 @@ def u_real_named(u_real):
 
 @pytest.fixture
 def atom_groups(u_real):
-    li_atoms = u_real.atoms.select
-    pf6_atoms = u_real.atoms.select
-    bn_atoms = u_real.atoms.select
-    fec_atoms = u_real.atoms.select_atoms()
+    li_atoms = u_real.atoms.select_atoms("type 22")
+    pf6_atoms = u_real.atoms.select_atoms("type 20").residues.atoms
+    bn_atoms = u_real.atoms.select_atoms("type 5").residues.atoms
+    fec_atoms = u_real.atoms.select_atoms("type 21").residues.atoms
     atom_groups = {"li": li_atoms, "pf6": pf6_atoms, "bn": bn_atoms, "fec": fec_atoms}
-
-
-def test_solvation_analysis_imported():
```

Comment:
I would leave the import test in, as it can't hurt for diagnostic purposes.

Reply:
I've created a test_basics.py file with the above code. I will try to add other thematically similar testing along with it. If the file ends up being very short, I may merge it back in later.